tests/auth-api.spec.js

```diff
@@ -0,0 +1,138 @@
+import { test, expect } from '@playwright/test';
+import { RegisterApi } from '../POM/modules/api/registerApi';
+import { LoginApi } from '../POM/modules/api/loginApi';
+import {
+  RESPONSE_MESSAGES,
+  generateUserCredentials,
+  ERRORS,
+} from '../fixtures';
+
+//Declare reusable variables
+let loginEmail, loginPassword, loginUsername;
+
+//Set serial execution
+test.describe.configure({ mode: 'serial' });
+
+test.describe('Register and login via Api', () => {
+  test('NE - Register with all input fields emty', async ({ page }) => {
```

Comment:
Typo - `empty`. I mali tip za pisanje test case-ova:
`Shouldn't be able to register without providing data`, da stavljas expected result u opis test case-a.